frontend/src/components/ViewTrips/trips-container.js

```diff
@@ -65,17 +66,21 @@ class TripsContainer extends React.Component {
         .where(DB.TRIPS_COLLABORATORS, 'array-contains', curUserUid)
         .orderBy(DB.TRIPS_UPDATE_TIMESTAMP, 'desc')
         .onSnapshot(querySnapshot => {
-          const tripsContainer = querySnapshot.docs.map(doc =>
+          const tripsContainer = querySnapshot.docs.map((doc, idx) =>
               ( <Trip
                   tripData={doc.data()}
                   tripId={doc.id}
                   handleEditTrip={this.props.handleEditTrip}
+                  eventKey={String(idx)}
```

Comment:
this can just be `eventKey={idx}` I believe

Reply:
That is what I tried originally and it actually does not work for some reason.